tests/RobotFramework/tests/tedge_connect/tedge_connect_test.robot

```diff
@@ -40,6 +40,14 @@ tedge_disconnect_test_sm_services
     Should Contain    ${output}    tedge-agent service successfully stopped and disabled!
     Should Contain    ${output}    tedge-mapper-c8y service successfully stopped and disabled!
 
+
+tedge cert upload c8y command fails
+    Execute Command    tedge config set c8y.root_cert_path /etc/ssl/certs_test    
+    ${output}=    Execute Command    sudo env C8YPASS\='password' tedge cert upload c8y --user ${C8Y_CONFIG.username}     ignore_exit_code=${True}    stdout=${False}    stderr=${True}
```

Comment:
Since we don't need real username/password I would also avoid using the ci username here.

```suggestion
    ${output}=    Execute Command    sudo env C8YPASS\='password' tedge cert upload c8y --user exampleuser     ignore_exit_code=${True}    stdout=${False}    stderr=${True}
```

Reply:
fixed